.travis.upgrade.docker.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Upgrades to docker engine 1.13.0 ( https://gist.github.com/dylanscott/ea6cff4900c50f4e85a58c01477e9473 )
```

Comment:
if docker 1.13.0 and higher is needed, could you pls change it also here?
https://github.com/hawkular/hawkinit/blob/5482a7e71f3758a6e8d5d270bad68103dadd3a42/lib/checkCommands.js#L13